winter_messaging_transactional/tests/tests/test_consumer.py

```diff
@@ -0,0 +1,261 @@
+import json
+from time import sleep
+from uuid import uuid4
+
+import pytest
+from testcontainers.rabbitmq import RabbitMqContainer
+
+from winter.core.json import JSONEncoder
+
+from winter_messaging_transactional.tests.conftest import event_consumer
+from winter_messaging_transactional.tests.helpers import EVENT_HANDLING_TIMEOUT
+from winter_messaging_transactional.tests.helpers import create_rabbitmq_connection
+from winter_messaging_transactional.tests.helpers import get_rabbitmq_url
+from winter_messaging_transactional.tests.helpers import read_all_inbox_messages
+from winter_messaging_transactional.tests.app_sample.dao import ConsumerDAO
+from winter_messaging_transactional.tests.app_sample.events import SampleEvent
+from winter_messaging_transactional.tests.app_sample.events import RetryableEvent
+from winter_messaging_transactional.consumer.inbox.inbox_message import InboxMessage
+from winter_messaging_transactional.consumer.inbox.inbox_message_dao import InboxMessageDAO
+from winter_messaging_transactional.producer.outbox import OutboxEventPublisher
+from winter_messaging_transactional.producer.outbox import OutboxMessage
+from winter_messaging_transactional.producer.outbox import OutboxMessageDAO
+from winter_messaging_transactional.tests.helpers import run_consumer
+from winter_messaging_transactional.tests.helpers import run_processor
+from winter_messaging_transactional.tests.helpers import wait_for_result
+
+
+def test_consume_without_error(database_url, rabbit_url, event_processor, injector, session):
+    event_publisher = injector.get(OutboxEventPublisher)
+    event_id = 1
+    payload = 'consume_without_error'
+
+    consumer_dao = injector.get(ConsumerDAO)
+
+    # Act
+    with event_consumer(database_url, rabbit_url, consumber_id='consumer_correct'):
+        event = SampleEvent(id=event_id, payload=payload)
+        event_publisher.emit(event)
+        session.commit()
+        consumed_event = wait_for_result(seconds=10, func=lambda: consumer_dao.find_by_id(event_id))
```

Comment:
looks like we can use 10 as default value for this function and don't set it explicitly in each case